From 61eda0a892a8ea8ad8292a51deb6dc0b70c9fa7e Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Thu, 2 Feb 2023 14:19:45 +0000
Subject: [PATCH] hwdb: update autosuspend db

(cherry picked from commit 93f9b9cc6de35d15a4dc09b5abe4f374aff0038c)

Related: RHEL-5950
---
 hwdb.d/60-autosuspend-fingerprint-reader.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
index 1af46fe079..a5cc89474f 100644
--- a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
+++ b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
@@ -71,7 +71,7 @@ usb:v08FFp5731*
  ID_PERSIST=0
 
 # Supported by libfprint driver aes4000
-usb:v5501p08FF*
+usb:v08FFp5501*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
